src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -1246,28 +1246,13 @@ private void UpdateImage(int key, PropertyValue value)
             {
                 if (_resourceUrl != null)
                 {
-                  Size2D originalImageSize = ImageLoading.GetOriginalImageSize(_resourceUrl);
-                  Size2D imageSize = originalImageSize;
-                  originalImageSize?.Dispose();
-
-                  int adjustedDesiredWidth, adjustedDesiredHeight;
-                  float aspectOfDesiredSize = (float)_desired_height / (float)_desired_width;
-                  float aspectOfImageSize = (float)imageSize.Height / (float)imageSize.Width;
-                  if( aspectOfImageSize > aspectOfDesiredSize)
-                  {
-                      adjustedDesiredWidth = _desired_width;
-                      adjustedDesiredHeight = imageSize.Height * _desired_width / imageSize.Width;
-                  }
-                  else
-                  {
-                      adjustedDesiredWidth = imageSize.Width * _desired_height/ imageSize.Height;
-                      adjustedDesiredHeight = _desired_height;
-                  }
-
-                  PropertyValue returnWidth = new PropertyValue(adjustedDesiredWidth);
+                  Size2D desiredSize = new Size2D(_desired_width,_desired_height);
```

Comment:
Could you check warning message?
Check warning on line 1249 in src/Tizen.NUI/src/public/BaseComponents/ImageView.cs
Check warning on line 1250 in src/Tizen.NUI/src/public/BaseComponents/ImageView.cs